db/seeds.rb

```diff
@@ -7,3 +7,9 @@
 broc.image.attach(io: File.open(Rails.root.join('app', 'assets', 'images', 'broccoli.png')), filename: 'broccoli.png', content_type: 'image/png')
 potato = Award.find_or_create_by(title: "Breezy", value: 40)
 potato.image.attach(io: File.open(Rails.root.join('app', 'assets', 'images', 'potato.png')), filename: 'potato.png', content_type: 'image/png')
```

Comment:
Tev vajag papildināt, ar condition, gadijumā jā attels jau ir, tad nepievienot to